pkg/collector/corechecks/ebpf/c/tcp-queue-length-kern.c

```diff
@@ -0,0 +1,145 @@
+#include <linux/compiler.h>
```

Comment:
To make it crystal clear that these files are runtime compiled, we've been putting them in a `runtime` subdirectory. So the path would be `pkg/collector/corechecks/ebpf/c/runtime/tcp-queue-length-kern.c`

Reply:
Done